metadata-integration/java/spark-lineage/src/main/java/datahub/spark/DatahubSparkListener.java

```diff
@@ -209,14 +212,6 @@ public Void apply(SparkContext sc) {
                 log.warn("Failed to close underlying emitter due to {}", e.getMessage());
               }
             }
-            consumers().forEach(x -> {
```

Comment:
was this deletion intentional? I thought we needed the `close` for Spark to release resources correctly

Reply:
The consumers() function no longer exists. This looks like it got refactored to be handled directly by the emitters.

Reply:
restored consumers